.changeset/rotten-flies-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query': patch
+---
+
+Observe `ValueSpecification` for query builder editors.
```

Comment:
Let's get rid of this, this comes as fixing problem coming from https://github.com/finos/legend-studio/issues/1037